Makefile

```diff
@@ -546,7 +546,7 @@ install-bin-libexec: $(BINLIBEXECLIST)
 
 install-configs: $(CONFIGS)
 	$(QUIET_INST)$(foreach f,$(CONFIGS),$(call INSTALL_CONFIG,$f,$(dir $(CONFIG_PATH))))
-	$(QUIET_INST)ln -sf $(DEFAULT_HYPERVISOR_CONFIG_PATH) $(CONFIG_PATH)
+	$(QUIET_INST)ln -sf $(DEFAULT_HYPERVISOR_CONFIG) $(DESTDIR)/$(CONFIG_PATH)
```

Comment:
I'm not sure this is correct. Previously, if I ran `make install DESTDIR=/tmp/james` it would do:

```sh
ln -sf /usr/share/defaults/kata-containers/configuration-qemu.toml /usr/share/defaults/kata-containers/configuration.toml
```

... but with this change, it does something like:

```sh
ln -sf configuration-qemu.toml /tmp/james//usr/share/defaults/kata-containers/configuration.toml
```

However, what we want is:

```sh
ln -sf /tmp/james//usr/share/defaults/kata-containers/configuration-qemu.toml /tmp/james//usr/share/defaults/kata-containers/configuration.toml
```

Reply:
@jodh-intel if we make the symlink to a absulute path in the case the the content of /tmp/james is moved to root (`/`) wont be a valid reference anymore. But if we do a relative link it will point the config we want all the time.

Reply:
But won't that only work if you create that relative link if you're `cd`'d to the directory where `configuration-qemu.toml` lives?

```sh
(cd $(dir $(DESTDIR)/$(CONFIG_PATH)) && ln -sf $(DEFAULT_HYPERVISOR_CONFIG) $(DESTDIR)/$(CONFIG_PATH))
```

That's starting to look like lisp ;)

Reply:
Why do we want absolute paths for symlinks? Shouldn't the location for the symlink file be consistent so the relative path is well defined?